timetagger/__main__.py

```diff
@@ -233,7 +234,7 @@ def load_credentials():
 
 
 def load_trusted_proxies():
-    return [s.strip() for s in config.proxy_auth_trusted.replace(";", ",").split(",")]
+    return iptools.IpRangeList(*[s.strip() for s in config.proxy_auth_trusted.replace(";", ",").split(",")])
```

Comment:
Fix style error (I think):
```suggestion
    ips = [s.strip() for s in config.proxy_auth_trusted.replace(";", ",").split(",")]
    return iptools.IpRangeList(*ips)
```